sql/core/src/main/scala/org/apache/spark/sql/SQLConf.scala

```diff
@@ -160,6 +162,9 @@ private[sql] class SQLConf extends Serializable {
 
   private[spark] def useSqlSerializer2: Boolean = getConf(USE_SQL_SERIALIZER2, "true").toBoolean
 
+  private[spark] def useJacksonStreamingAPI: Boolean =
```

Comment:
Can you add comment to explain that it is a temporary flag and we will remove the old code path in 1.5?